packages/site-admin/README.md

```diff
@@ -1,46 +1,193 @@
+Maintaining a visually and functionally consistent experience with the WordPress admin is a priority. 
+
+The package is designed as a reliable foundation for administrative applications, ensuring a cohesive and intuitive experience across different projects.
 
 ### Minimal External Dependencies
 
-- Relies solely on WordPress dependencies whenever possible.
-- Avoids integrating with external frameworks unless strictly necessary due to Core limitations.
-- While included in this Calypso monorepo for hosting and centralization, it is not tightly coupled with its libraries.
+- Leverages WordPress dependencies whenever possible.
+- Avoids integrating external frameworks unless absolutely necessary due to Core limitations.
+- Hosted within the Calypso monorepo for centralization but remains independent of its libraries.
+
+## Getting Started: Building Your Admin UI
+
+This section provides a basic guide to creating a complete admin interface using the components provided by this package.
+
+The following example demonstrates how navigation is structured within the sidebar.
+It includes a sidebar layout, individual navigation items, and a routing context to manage navigation state dynamically.
+
+## 1. Navigation System
+
+### 1.1 The `RouterProvider` Component
+
+At the core of the navigation system is the `RouterProvider` component,
+which establishes a routing context to manage navigation state
+throughout your application.
+
+```tsx
+<RouterProvider routes={ [your-routes-here] }>
+  { /* Your application content */ }
+</RouterProvider>
+```
+
+The `routes` property is an array of route definitions that map paths to handlers. The optional `pathArg` property defines the URL query parameter that stores the current path (defaults to `p` if not specified).
+
+## 2. The App Sidebar
+
+### 2.1 Defining Routes
+
+Before creating navigation elements, we need to define the routes that our application will use. Each route must have a unique name, a path, and optionally areas and width configurations.
+
+```tsx
+// Define our application routes
+const appRoutes = [
+  {
+    name: 'reports',
+    path: '/reports',
+    areas: {
+      content: <ReportsScreen />
+    },
+    widths: {
+      content: 100
+    }
+  },
+  {
+    name: 'settings',
+    path: '/settings',
+    areas: {
+      content: <SettingsScreen />
+    },
+  },
+  {
+    name: 'archive',
+    path: '/archive',
+    areas: {
+      content: <ArchiveScreen />
+    },
+  }
+];
+```
+
+### 2.2 Defining Sidebar Content
+
+The `eSidebarContent` component renders navigation items within the sidebar. Each item corresponds to a specific route, and the `useLocation` hook determines whether it is active.
+
+```tsx
+const SidebarContent = () => {
+    const { path } = useLocation();
+
+    return (
+        <ItemGroup>
+            <SidebarNavigationItem
+                icon={ <Icon icon={ navigation } /> }
```

Comment:
```suggestion
                icon={ navigation }
```

Reply:
I can still see places in this file where the `icon` prop of `SidebarNavigationItem` is being passed an `<Icon />` component, instead of just the icon.